src/main/java/ticTacToe/grid/Grid.java

```diff
@@ -84,6 +86,29 @@ public boolean isFinishedGame(Lines lines) {
         return allOccupiedCells() || lines.isWinning(this);
     }
 
+    public Mark markThatMoves() {
+        makeMarksCount();
```

Comment:
This method call is a bit tricky.
Its not obvious what impact it has on the remaining code, as it does not seem to produce any outputs/return values.
The way I see it "communicates" with the remaining if-statements is through the instance variables "crossesCount" and "noughtsCount". While it works, its not the most obvious and intention-revealing way of dealing with it.
I would suggest a bit of code like
```
public Mark markThatMoves() {
    int crossesCount = countMarks(CROSS);
    int noughtsCount = countMarks(NOGHT);
    ...
}
```
This way we understand the relationship between these pieces of information and where they come from.